eahub/base/static/components/search-profiles/search.html

```diff
@@ -145,7 +145,7 @@ <h5 class="modal-title">Filters</h5>
                   >
                     <div class="row search__hit-summary" v-if="item.summary">
                       <h5>Summary</h5>
-                      <ais-snippet :hit="item" attribute="summary"/>
+                      <ais-snippet :hit="item" attribute="summary"/><!-- TODO roland: replace newlines with <br>? -->
```

Comment:
FYI I think that isn't possible, algolia doesn't allow spaces in snippets, that's part of their indexing engine tech specs. And there can't be HTML tags in those fields. Could you remove those TODOs then?

Reply:
Can you please clarify, what do you mean "doesn't allow spaces in snippets"? Would newline characters, tab characters or non-breaking space characters be allowed? Maybe we could use them in place of HTML <br> tags? And then during rendering replace them with <br> tags?